cli/util/file.go

```diff
@@ -2,17 +2,16 @@ package util
 
 import (
 	"io/ioutil"
-	"log"
 	"os"
 )
 
 // GetFileContents reads a file and returns its content as a string
```

Comment:
Is there something like JavaDocs for Go?

Reply:
Yes there is, although its not as good as JavaDoc. But Go has an useful feature which warns you about undocumented public functions out of the box when using Go. Due to that reason, all public functions have a GoDoc comment attached.